src/extension/documenter.helper.ts

```diff
@@ -59,19 +59,19 @@ export default {
   },
 
   /**
-   * Get a date formatted according to the format set in the extension's settings under DateFormat.
+   * Get a date formatted according to the format set in the extension's settings under Locale and DateTimeFormat.
    */
-  getFormattedDate(): string {
+  getFormattedDateTime(): string {
```

Comment:
Using locale is an interesting idea, but you're not returning a DateTime - there's no time included in the return value.